### PR TITLE
fix status-indicator links & linting errors

### PR DESCRIPTION
**Pull request by halocline**
#### What does this PR do?

- Fixes broken links to the Status Indicators page from [Notification](https://deploy-preview-1814--keen-mayer-a86c8b.netlify.app/components/notification) and [Toast Notifications](https://deploy-preview-1814--keen-mayer-a86c8b.netlify.app/templates/toast-notifications)
- Also applies linter rules which weren't applied/checked post-commit / pre-merge

### CODE REVIEW

**Comment by halocline** [comment]
@amandadupell tagging you for review -- just added you as a contributor, so need to accept that invite before I can tag you properly.

**Review by britt6612 — APPROVED** [review]
LGTM!  I just ran into `eslint` issues on my side we should connect with  @amandadupell make sure she has correct linting

File: aries-site/src/examples/cardPreviews/toast-notification.js [hunk 1]
```diff
@@ -36,3 +37,9 @@ export const ToastPreview = ({card, message, title}) => (
     </Box>
   </Box>
 );
+
+ToastPreview.propTypes = {
+  card: PropTypes.string,
```
Comment (amandadupell) [inline aries-site/src/examples/cardPreviews/toast-notification.js:42]:
```suggestion
  card: PropTypes.boolean,
```
was using this as a boolean flag

**Review by amandadupell — COMMENTED** [review]
one small comment and curious to know why 2 checks are currently failing, was my `yarn.lock` file out of date?

**Review by taysea — APPROVED** [review]

**Pull request merged**